apps/condo/domains/ticket/utils/helpers.ts

```diff
@@ -649,4 +649,23 @@ export function hasUnreadResidentComments (lastResidentCommentAt, readResidentCo
     }
 
     return false
+}
+
+/** analyticsData array validation function. 
+ * The array can only be empty on the first render. Even if there are no tickets, mock objects are placed in analyticsData. 
+ * For the correct operation of the unload button in Excel, it is necessary to check the objects nested in the array for the presence of at least one ticket */
+export function isEmptyAnalyticsData (analyticsData) {
+    if (analyticsData === null) {
+        return true
+    } else {
+        let count = 0
+        for (let i = 0; i < analyticsData.length; i++) {
```

Comment:
There is no need to check every element - this increases the complexity of the algorithm. Try to reduce the number of operations